docs/rules/README.md

```diff
@@ -17,5 +17,5 @@
 | Rule ID | Description |    |
 |:--------|:------------|:---|
 | [@intlify/vue-i18n/<wbr>no-dynamic-keys](./no-dynamic-keys.html) | disallow localization dynamic keys at localization methods |  |
-| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys |  |
+| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys | :black_nib: |
```

Comment:
```suggestion
| [@intlify/vue-i18n/<wbr>no-unused-keys](./no-unused-keys.html) | disallow unused localization keys | :black_nib: **(experimental)** |
```

Reply:
@ota-meshi 
I've added `experimental`  word to here.

Reply:
@kazupon 
This list is generated by the `generate` script, how would you like to add it?

- Add the `experimentalFixable` property to `mea.docs` of `no-unused-keys` rule.
- Change the `update-docs-index` to add words only for the `no-unused-keys` rule.
- Other?

Reply:
@ota-meshi 
Ohh, I forgot it...
Don't worry about it. 😅